libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.h

```diff
@@ -38,6 +38,17 @@ typedef NS_ENUM(NSInteger, SFRestMethod) {
     SFRestMethodPATCH
 } NS_SWIFT_NAME(RestRequest.Method);
 
+/**
+ * HTTP methods for requests.
+ */
+typedef NS_ENUM(NSInteger, SFSDKNetworkServiceType) {
```

Comment:
Would it make sense for this to live in `SFNetwork` instead since it's more of a network property than API client related?

Reply:
This is part of NSURLRequest in foundation. Hence it made more sense to add it our SFRestRequest. i.e. easier to lookup and use for a developer.

Reply:
Sounds good.